src/test/java/com/sonyericsson/rebuild/RebuildValidatorTest.java

```diff
@@ -393,6 +398,110 @@ public void testRebuildSupportedUnknownParameterValue() throws Exception {
 				page.asText().contains("This is a mark for test"));
 	}
 
+	public void testWhenProjectWithExistingParamAndOverridingParamWhenRebuild()
+			throws Exception {
```

Comment:
I would suggest to rename this test, I think there's something wrong with the sentence itself and it is not really clear what's the goal of the test, probably something like "newParametersShouldOverrideExistingPatametersIfHaveSameName" or something like that.